src/networking.c

```diff
@@ -22,6 +22,11 @@
 #include <math.h>
 #include <ctype.h>
 
+/* Threshold for reusing client arguments. new client argument is created
+ * if the requested argument size is exceeds the current size, or if it
```

Comment:
feel a little confused for this comment.
```suggestion
 * if the requested argument size exceeds the current size, or if it
```

Reply:
removed it, feel it ugly too.